crates/rpc/src/handlers/block.rs

```diff
@@ -9,22 +10,9 @@ use warp::{
     reply::{Reply, with_status},
 };
 
-use crate::types::{
-    errors::ApiError,
-    id::ID,
-    response::{BeaconResponse, RootResponse},
-};
-
-/// Called by `/blocks/<block_id>/root` to get the Tree hash of the Block.
-pub async fn get_block_root(block_id: ID, db: ReamDB) -> Result<impl Reply, Rejection> {
-    let block_root = get_block_root_from_id(block_id, db).await?;
-    Ok(with_status(
-        BeaconResponse::json(RootResponse { root: block_root }),
-        StatusCode::OK,
-    ))
-}
```

Comment:
don't we need this function for our get block root endpoint?

Reply:
Is this endpoint implemented on master? `/blocks/<block_id>/root`
I am not able to find this on the master branch.

Reply:
It was accidentally removed in a previous PR, could you add it back in this PR

```
    let block_root = beacon_base
        .and(path("blocks"))
        .and(param::<ID>())
        .and(path("root"))
        .and(end())
        .and(get())
        .and(db_filter.clone())
        .and_then(move |block_id: ID, db: ReamDB| get_block_root(block_id, db))
        .with(log("block_root"));
```

Reply:
Added this